link/platform.go

```diff
@@ -0,0 +1,19 @@
+package link
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func platformPrefix(symbol string) string {
+
+	prefix := runtime.GOARCH
+	switch prefix {
+	case "i386":
+		prefix = "ia32"
+	case "amd64":
+		prefix = "x64"
+	}
```

Comment:
This branch/PR is targeting arm64, but I don't see an arm64 entry here. :) Note that according to https://github.com/golang/go/blob/master/src/go/build/syslist.go there's also an `arm64be` entry, but the Go compiler doesn't seem to support that out of the box:
```
λ  ~/documents/go/sandbox  GOARCH=arm64be go build
cmd/go: unsupported GOOS/GOARCH pair linux/arm64be
```
so I think we can skip an arm64be entry.

Reply:
Yes, my approach was to "patch" only for those platform that differ from GOARCH. Maybe it is better to make these translations more explicint in map?

Reply:
The structure is fine as you suggested, I would just add a default case with `return symbol`.